policies/roadmap.md

```diff
@@ -0,0 +1,70 @@
+# OpenSSL project roadmap
+
+### Last modified 21 March 2022
+
+This document is the OpenSSL project roadmap. It is a living document
+and is expected to change over time. Objectives and dates should be
+considered aspirational.
+
+## Objectives
+
+Some of these objectives can be achieved more easily and quickly
+than others.
+
+#### Regular releases
+
+The OpenSSL project wants to move to having a regular release interval
+of approximately six months.
```

Comment:
This section isn't really roadmap material, is it?  It looks more like a release strategy, which would be a different document...

Reply:
Making it policy would mandate the cycle.  At this stage it's still aspirational.

I'm, fine removing this section if it doesn't belong here.

Reply:
[nudge nudge]
I really think it doesn't belong here